### PR TITLE
ENH: Added script to search keywords in all of SCILPY scripts

### PR DESCRIPTION
**Pull request by ppoulin91**

### CODE REVIEW

File: scripts/scil_find_script.py [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+    parser = ArgumentParser(description=__doc__,
+                            formatter_class=RawTextHelpFormatter)
+    parser.add_argument('keywords', nargs='+',
+                        help="Search keywords")
+    return parser
+
+
+def main():
+    parser = _build_arg_parser()
+    add_verbose_arg(parser)
```
Comment (EmmaRenauld) [inline scripts/scil_find_script.py]: Should go in _build_arg_parser I think

File: scripts/scil_find_script.py [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+            print(filename, '"{}"'.format(display_info))
+
+    if not matches:
+        print("No results found!")
+
+
+def _is_matching_keywords(keywords, texts):
+    """Test multiple texts for matching keywords. Returns True only if all
+    keywords are present in any of the texts.
+
```
Comment (EmmaRenauld) [inline scripts/scil_find_script.py]: I'm not sure "is" is the good name.

_find?

File: scripts/scil_find_script.py [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+def _build_arg_parser():
+    parser = ArgumentParser(description=__doc__,
+                            formatter_class=RawTextHelpFormatter)
+    parser.add_argument('keywords', nargs='+',
+                        help="Search keywords")
+    add_verbose_arg(parser)
+    return parser
+
+
+def main():
```
Comment (jhlegarreta) [inline scripts/scil_find_script.py]: Is there an agreed convention on the order of the methods in the scripts, e.g.  `_build_arg_parse`r and `main` always first, then the rest of methods, or else are the majority of the scripts making these two be last methods?

Comment (ppoulin91) [reply scripts/scil_find_script.py]: I would guess that `build_arg_parser` should come first, but I don't think there is any convention for the rest

Comment (jhlegarreta) [reply scripts/scil_find_script.py]: OK. I guess having one would improve the readability of the scripts, but it should not be a show-stopper.

File: scripts/scil_find_script.py (hunk 2 above)
Comment (ppoulin91) [reply scripts/scil_find_script.py]: It doesn't return aything, son I don't think '_find' is the right one. What do you think of '_test_matching_keywords'?

**Comment by ppoulin91** [comment]
@jhlegarreta Yes, I would be nice to add more options if necessary in the future

**Comment by arnaudbore** [comment]
Add a test please.

**Comment by arnaudbore** [comment]
Tested: 
```
scil_find_script.py convert
No results found!
```

**Comment by mdesco** [comment]
On my mac, same thing as @arnaudbore . Cannot find anything, no matter what keyword I enter. 

scil_find_script.py quick
No results found!

scil_find_script.py bundle
No results found!

scil_find_script.py track
No results found!

**Comment by ppoulin91** [comment]
I overall made it nicer to look at, and use bold instead of color when printing to support different terminal configurations. We also now detect when a script will not run and fall back to the docstring in that case.

I fixed the script directory, could @arnaudbore or @mdesco test it now?

**Comment by arnaudbore** [comment]
It works ! Now, can you add a test and change all your prints to logging. 
I may be wrong but I feel like the final print is a little bit too much.
You could have logging.info by default that print the different scripts names and when using verbose (i.e logging.debug) it would also show you the docstring.

**Comment by ppoulin91** [comment]
@arnaudbore Ready for a last check!

**Pull request merged**